fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchIncludeTest.java

```diff
@@ -7,6 +7,7 @@
 package com.ibm.fhir.server.test;
 
 import static com.ibm.fhir.model.type.String.of;
+import static org.junit.Assert.assertNull;
```

Comment:
convert to testng (per the prior example in the review)

Reply:
Done.